moviepy/video/io/ImageSequenceClip.py

```diff
@@ -117,20 +117,20 @@ def make_frame(t):
                 index = find_image_index(t)
 
                 if index != self.last_index:
-                    self.last_image = imread(self.sequence[index])[:, :, :3]
+                    self.last_image = imread(self.sequence[index], index=0)[0][:, :, :3]
```

Comment:
```suggestion
                    self.last_image = imread(self.sequence[index], index=0)[:, :, :3]
```

This looks a bit like we can push down the entire operation to ImageIO, but I lack context of the class itself. Does `last_image` mean that you wish to read the last image in the file?